src/main/java/com/virtudoc/web/service/SecurityService.java

```diff
@@ -0,0 +1,6 @@
+package com.virtudoc.web.service;
+
+public interface SecurityService {
+    boolean isAuthenticated();
+    void autoLogin(String username, String password);
```

Comment:
Same as other comment, these functions are already handled by the Spring Boot Security framework in #25. We should use SBSF whenever we can because it automatically handles things like password hashing, routing-level role-based filtering, and strong client-side cookie protections.